File: OrderOfOperationsDiv2/solve/cpp/OrderOfOperationsDiv2Solver.cpp
#include <fstream>
#include <topcoder>
#include "OrderOfOperationsDiv2.cpp"
namespace tc = TopCoder;

int main(int argc, char const *argv[]) {
  try {
    std::ifstream ifs(argv[1]);
    vector<string> s; tc::read(ifs, s);
    ifs.close();

    std::ofstream ofs(argv[2]);
    OrderOfOperationsDiv2 solver;
    tc::show(ofs, solver.minTime(s));
    ofs.close();
  } catch (std::exception &e) {
    std::cerr << e.what() << std::endl;
  }
  return 0;
}

File: OrderOfOperationsDiv2/solve/java/OrderOfOperationsDiv2Solver.java
import java.io.FileReader;
import java.io.FileWriter;

import java.util.List;
import java.util.ArrayList;

import org.topcoder.TopcoderReader;
import org.topcoder.TopcoderWriter;
import org.topcoder.TypeRef;

public class OrderOfOperationsDiv2Solver {
    public static void main(String[] args) {
    try {
        TopcoderReader reader = new TopcoderReader(new FileReader(args[0]));
        List<String> sBoxed = (List<String>) reader.next(new TypeRef<List<String>>(){}.getType());
        String[] s = new String[sBoxed.size()];
        for (int _i = 0; _i < sBoxed.size(); ++_i)
            s[_i] = sBoxed.get(_i);
        reader.close();

        OrderOfOperationsDiv2 solver = new OrderOfOperationsDiv2();
        TopcoderWriter writer = new TopcoderWriter(new FileWriter(args[1]));
        writer.write(solver.minTime(s));
        writer.close();
    } catch (Exception err) {
        err.printStackTrace(System.err);
    }
    }
}
